packages/client/test/miner/ethash.spec.ts

```diff
@@ -110,7 +110,10 @@ tape('start client', async (t) => {
   const client = await setupPowDevnet(minerAddress, true)
   t.ok(client.started, 'client started successfully')
   await stopClient(client, t)
-  const restartedClient = await setupPowDevnet(minerAddress, false)
-  await restartClient(restartedClient, t)
+  await client.open()
+  await ((client.service('eth') as FullEthereumService).execution as any).stateDB!.open()
```

Comment:
In general this is a great test too, so if we stop the client some internal variables are not cleaned up (so we cannot restart correctly). But this is a different issue, should be addressed though.